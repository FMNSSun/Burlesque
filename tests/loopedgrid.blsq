riro^^bc?-{)ab?i}m[sp