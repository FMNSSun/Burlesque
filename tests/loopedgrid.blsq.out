1 2 3 4 5 6 7 8 9
2 1 2 3 4 5 6 7 8
3 2 1 2 3 4 5 6 7
4 3 2 1 2 3 4 5 6
5 4 3 2 1 2 3 4 5
6 5 4 3 2 1 2 3 4
7 6 5 4 3 2 1 2 3
8 7 6 5 4 3 2 1 2
9 8 7 6 5 4 3 2 1